roda-ui/roda-wui/src/main/resources/config/theme/DisposalConfirmationDescription_sv_SE.html

```
@@ -1,2 +1,2 @@
-<p>På den här sidan kan du se de gallringsbekräftelser som har skapats för detta arkiv. En gallringsbekräftelse innehåller en rapport som samlar de logiska enheterna och extra metadatainformation. För att gallra den logiska enhet som är associerat med gallringsbekräftelsen, måste du explicit utföra gallringsåtgärden. 
-Efter gallring kan den logiska enheten antingen återställas eller tas bort permanent. </p>
+<p>På den här sidan kan du se de gallringsbekräftelser som har skapats för detta arkiv. En gallringsbekräftelse innehåller en rapport som samlar de förvaringsenheterna och extra metadatainformation. För att gallra den förvaringsenhet som är associerat med gallringsbekräftelsen, måste du explicit utföra gallringsåtgärden. 
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)